roles/nginx/defaults/main.yml

```diff
@@ -1,7 +1,8 @@
 ---
-logs_root: /var/log/nginx
+nginx_logs_root: /var/log/nginx
 nginx_user: www-data
 strip_www: true
+nginx_fastcgi_buffers: 8 8k
```

Comment:
I don't like that URL. It says to do `32 32k` which is 1MB. They don't see to understand it multiplies them.

Should add this one: https://gist.github.com/magnetikonline/11312172